### DISCUSSION TITLE
translation into Dutch

### DISCUSSION THREAD

**Discussion by 1mfaasj** [outline/outline#4119] [General]
I use Outline quite a lot (self-hosted) and I am willing to make a complete translation into Dutch for you.
are you open to this?

**Comment by tommoor** [comment]
It's 90% complete, perhaps you could take a pass at the translations so far.

https://translate.getoutline.com/proofread/outline/all/en-nl?filter=basic&value=5

**Reply by tommoor** [reply]
Dutch translations will be in the next release, https://github.com/outline/outline/pull/4120

**Reply by 1mfaasj** [reply]
I just completed the last translations. Thanks for upcoming update! When can I expect it?

**Reply by tommoor** [reply]
Community edition is released monthly. Cloud edition will be updated within 24h
